examples/customPlot.py

```diff
@@ -56,23 +56,16 @@ def updateRange(self, vb, viewRange):
         self._range = viewRange
         
         for pos in self.all_ticks:
-        
-            if pos not in self.visible_ticks:
-                self.visible_ticks[pos] = self.addTick(pos, movable=False, color="333333")
-            
-            tick = self.visible_ticks[pos]
-            
             tickValueIncludingPadding = (pos - viewRange[0]) / (viewRange[1] - viewRange[0])
             tickValue = (tickValueIncludingPadding*lengthIncludingPadding - origin) / length
             
-            visible = tickValue >= 0 and tickValue <= 1
+            visible = bool(tickValue >= 0 and tickValue <= 1)
```

Comment:
This should already be a `bool` without having to cast it, no?

Reply:
It's a `np.bool_` because of the arithmethics with `viewRange` above. And `PyQt5` throws a warning when one uses `np.bool_` instead of `bool`... Wasn't intuitive for me either. Maybe adding a comment would help?

Reply:
ahh; yeah that got me....yeah definitely add a comment for that!

Reply:
Added a comment, is that understandable?

Reply:
LGTM